app-misc/socnetv/files/socnetv-3.1-deps.patch

```diff
@@ -0,0 +1,10 @@
+--- a/socnetv.pro
```

Comment:
The patch needs some explanation/annotation. Why are we doing this and why is it okay to do?

Reply:
src/parser.h calls two classes provided by dev-qt/qtbase[-xml] :
QXmlStreamReader
QXmlStreamAttributes
Patch commented